lib/src/json_schema/json_schema.dart

```diff
@@ -83,6 +84,8 @@ class JsonSchema {
     Map<String, JsonSchema> refMap,
     RefProvider refProvider,
     Map<Uri, bool> metaschemaVocabulary,
+    CustomVocabulary customKeyword,
+    Map<String, Map<String, SchemaPropertySetter>> customVocabMap,
```

Comment:
Wait, don't we want to take `KeywordProcessor`s here?

Reply:
They `KeywordProcessor`s have been transform into something that works with the vocabulary maps. It happens in the `_createCustomVocabMap` function.